# HG changeset patch
# User Andreas Weber <andy.weber.aw@gmail.com>
# Date 1427657724 -7200
#      Sun Mar 29 21:35:24 2015 +0200
# Node ID d4d3bb493bcfbe9a15127130bfa97eb4e2a49061
# Parent  26a4ed0c9c5b2251f2d1e5d539ac3db80f705276
contributors.in: Adding Armin Müller for german translation

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -208,16 +208,17 @@ Ed Meyer
 Thorsten Meyer
 Stefan Miereis
 Petr Mikulik
 Mike Miller
 Serviscope Minor
 Stefan Monnier
 Antoine Moreau
 Kai P. Mueller
+Armin Müller
 Hannes Müller
 Victor Munoz
 PrasannaKumar Muralidharan
 Iain Murray
 Carmen Navarrete
 Todd Neal
 Philip Nienhuis
 Al Niessner
